charts/ceph-csi-cephfs/templates/nodeplugin-clusterrole.yaml

```diff
@@ -1,18 +0,0 @@
-{{- if .Values.rbac.create -}}
-{{- if .Values.topology.enabled }}
-kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: {{ include "ceph-csi-cephfs.nodeplugin.fullname" . }}
-  labels:
-    app: {{ include "ceph-csi-cephfs.name" . }}
-    chart: {{ include "ceph-csi-cephfs.chart" . }}
-    component: {{ .Values.nodeplugin.name }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-rules:
-  - apiGroups: [""]
```

Comment:
clusterrole binding still needs to be removed?

Reply:
oh.. yeah.. removed it @Madhu-1